public/app/plugins/panel/graph/graph_tooltip.ts

```diff
@@ -257,7 +257,7 @@ export default function GraphTooltip(this: any, elem: any, dashboard: any, scope
 
         series = seriesList[hoverInfo.index];
 
-        value = series.formatValue(sanitize(hoverInfo.value));
+        value = series.formatValue(hoverInfo.value);
```

Comment:
I don't follow why you removed the call to `sanitize` here

Reply:
I added it in my prior commit but now when I revisited the code I understood it as this isn't a user input. Its the data value, right?

Reply:
it is user input (colors are set by users, can also come from data)

Reply:
OK ok, so that applies for the `hoverInfo.value` as well?

Reply:
value is from data

Reply:
so also good to sanitize :)